docs/troubleshooting.md

```diff
@@ -315,10 +315,10 @@ Note that the `[syscall]` suffix in a metric name means that it is instanciated
 
 Note that if ProcfsScraper is unable to open /proc it is not able to open any of the subdirectories, but only procfs_could_not_open_proc_dir will be incremented in that case.
 
-### Falco timers per syscall
+### system_inspector timers per syscall
```

Comment:
It's probably worth mentioning somewhere before this point, that system_inspector in the current incarnation is actually nothing more than Falco inspector.

Reply:
Makes sense, I'll find a place for it.